contracts/zero-ex/tests/WrapEth.t.sol

```diff
@@ -33,7 +33,9 @@ contract WrapEth is Test, ForkUtils, TestUtils {
     DeployZeroEx.ZeroExDeployed zeroExDeployed;
 
     function setUp() public {
-        zeroExDeployed = new DeployZeroEx().deployZeroEx();
+        zeroExDeployed = new DeployZeroEx(
```

Comment:
We don't have overloaded constructors in solidity, but maybe some sort of utility function somewhere that just constructs DeployZeroEx with all zeros for the config would be nice. All the DeployConfig code really is awkwardly bolted onto DeployZeroEx, I was just shooting for something quick with minimal changes to DeployZeroEx so that we could get the tests we needed running. Definitely can and should be cleaned up, probably should loop @dextracker into that conversation.